CONTRIBUTING.md

```diff
@@ -239,7 +239,10 @@ consensus to merge a pull request (remember that discussions may have been
 spread out over GitHub, mailing list and IRC discussions). The following
 language is used within pull-request comments:
 
-  - ACK means "I have tested the code and I agree it should be merged";
+  - ACK means "I have tested the code and I agree it should be merged", involving
```

Comment:
thanks!
also maybe say (t)ACK here, as people use tACK, tested ACK and ACK interchangeably with the same intent